src/my_plugin/my_plugin_android/android/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 34
```

Comment:
With all of these changes should we also be looking at the `minSdkVersion` as well?

Reply:
I always keep the min SDK as low at the other dependencies allows it.